opencsp/test/data/sofast_common/Ensemble_split_NSTTF_facet.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
consider filing a follow-on issue to randomly generate this test data at runtime.

Reply:
Let's discuss this. Even if it is a more robust test, it may not be straight forward or practical to generate random values.